cmd/litcli/ln.go

```diff
@@ -0,0 +1,751 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"crypto/rand"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/lightninglabs/taproot-assets/asset"
+	"github.com/lightninglabs/taproot-assets/rfqmsg"
+	"github.com/lightninglabs/taproot-assets/taprpc"
+	"github.com/lightninglabs/taproot-assets/taprpc/rfqrpc"
+	tchrpc "github.com/lightninglabs/taproot-assets/taprpc/tapchannelrpc"
+	"github.com/lightningnetwork/lnd/cmd/commands"
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/lightningnetwork/lnd/lnrpc/routerrpc"
+	"github.com/lightningnetwork/lnd/lntypes"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/lightningnetwork/lnd/record"
+	"github.com/urfave/cli"
+)
```

Comment:
just skimmed this commit really quickly but it isnt yet immediately obvious to me why this is here and not in the taproot assets `cmd` directory? ie what makes this different from `loopcli` or equivalent?

Reply:
We do eventually want the logic that's done in this file be RPCs in `tapd`, then it will make sense to move them to `tapcli`. But for now we need access to both `lnd`'s and `tapd`'s RPCs from within a single command, which is done most conveniently in `tapcli` (see the way we bake a macaroon just to connect to `tapd`).